src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: Props) => {
+  const validTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
Technically, it's a valid tab, however, what if we call it current tab or just selectedTab? 
This may be more accurate.